### PR TITLE
fix(iam_inline_policy_no_administrative_privileges): set resource id as the entity name

### PR DESCRIPTION
**Pull request by MrCloudSec**
### Description

Set resource id as the entity name.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by toniblyx — DISMISSED** [review]
LGTM

File: prowler/providers/aws/services/iam/iam_inline_policy_no_administrative_privileges/iam_inline_policy_no_administrative_privileges.py [hunk 1]
```diff
@@ -10,7 +10,7 @@ def execute(self) -> Check_Report_AWS:
                 report = Check_Report_AWS(self.metadata())
                 report.region = iam_client.region
                 report.resource_arn = policy.arn
-                report.resource_id = policy.name
+                report.resource_id = policy.entity
```
Comment (jfagoagas) [inline prowler/providers/aws/services/iam/iam_inline_policy_no_administrative_privileges/iam_inline_policy_no_administrative_privileges.py:13]: Why this change? Please include an explanation in the PR body.

Thanks!

Comment (MrCloudSec) [reply prowler/providers/aws/services/iam/iam_inline_policy_no_administrative_privileges/iam_inline_policy_no_administrative_privileges.py:13]: Because the inline policy it is not an unique resource and it may lead to duplicate finding IDs.

Comment (jfagoagas) [reply prowler/providers/aws/services/iam/iam_inline_policy_no_administrative_privileges/iam_inline_policy_no_administrative_privileges.py:13]: I didn't see the "inline" part, totally agree then. Good catch!!

**Review by jfagoagas — APPROVED** [review]
🔝
